saleor/settings.py

```diff
@@ -620,8 +620,8 @@ def get_host():
     RAZORPAY: {
         "module": "saleor.payment.gateways.razorpay",
         "config": {
-            "store_card": False,
-            "auto_capture": None,
+            "store_card": get_bool_from_env("RAZORAY_STORE_CARD", False),
+            "auto_capture": get_bool_from_env("RAZORPAY_AUTO_CAPTURE", None),
```

Comment:
I don't think we should provide it for razorpay as it doesn't support authorization. It will always auto capture.

Reply:
We should keep consistent configuration across gateways.